roles/openshift_node/tasks/main.yml

```diff
@@ -34,6 +34,32 @@
         dns_ip: "{{ openshift_dns_ip | default(none) | get_dns_ip(hostvars[inventory_hostname])}}"
         env_vars: "{{ openshift_node_env_vars | default(None) }}"
 
+# https://docs.openshift.com/container-platform/3.4/admin_guide/overcommit.html#disabling-swap-memory
+- name: Check for swap usage
+  command: grep swap /etc/fstab
```

Comment:
`grep -v "^#" /etc/fstab | grep swap` or something better to avoid false positives on commented lines?